plugins/modules/get_certificate.py

```diff
@@ -154,8 +155,8 @@
 
 from distutils.version import LooseVersion
 from os.path import isfile
-from socket import setdefaulttimeout, socket
-from ssl import get_server_certificate, DER_cert_to_PEM_cert, CERT_NONE, CERT_OPTIONAL
+from socket import create_connection, setdefaulttimeout, socket
+from ssl import DER_cert_to_PEM_cert, CERT_NONE, CERT_OPTIONAL
```

Comment:
```suggestion
from ssl import get_server_certificate, DER_cert_to_PEM_cert, CERT_NONE, CERT_OPTIONAL
```
Otherwise the 2.6 code path won't work :)

Reply:
yep